site/blog_collections/_running/2025-12-31-2025_running_year_in_review.md

```diff
@@ -0,0 +1,93 @@
+
+{% capture table_content %}
+|------------|------------|
+| <img src="/img_running/2025-12-31/strava1.jpg" data-src="/img_running/2025-12-31/strava1.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> | <img src="/img_running/2025-12-31/strava2.jpg" data-src="/img_running/2025-12-31/strava2.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> |
+{% endcapture %}
+{{ table_content | markdownify }}
+
+The volume is slightly less than 2024 (5000K / 501h) - primarily due to Tuesday cross training in the first half of the year (stair stepper, prep for Mozart). I think this cross training (and added strength training in the first half year in particular) was well invested. I had an incredible Mozart 100 race: felt super strong compared to my UTMB CCC experience in 2024. Much stronger hiking and less critical performance drop.
+
+# Races
+
+In 2025, I completed `16 races` totaling `1055K` with `18500m` of elevation gain over `130h` of racing. The races were split between `6 ultras` (≥50km) and `10 non-ultras`, as well as `9 trail races` and `7 road races`.
+
+Highlights include my longest race ever: [Kullamannen 172.7K](/blog/running/2025/11/18/kullamannen_by_utmb_2025) with `26h41m` on feet. I also achieved a backyard ultra PB of `20 yards` (`134K`) at [Wayout Backyard Ultra](/blog/running/2025/04/01/race_wayout_backyard_ultra), finishing 13th of 163 starters.
+
+# Half's
+This year I chased the sub-90 minute half marathon goal across three races. The results were bittersweet - while my Strava times consistently showed sub-90 performances, the official chip times were all just above the mark. Close, but not quite there! Here are the three attempts:
+
+- [Race: Cross Island Bornholm](/blog/running/2025/04/19/race_cross_island_bornholm) - Official: 1h30m10s, Strava half: 1h29m39s 😆
+- [Race: Odense City Half](/blog/running/2025/05/21/race_odense_city_half) - Official: 1h31m, Strava half: 1h29m46s 😭
+- [Race: Bestseller Aarhus City Half](/blog/running/2025/06/27/race_bestseller_aarhus_city_half) - Official: 1h30m37s, Strava half: 1h29m50s
+
+I'm really happy with these results even though I didn't manage to set a PB for the half. I feel much faster overall than in 2024 - but still wasn't able to beat my own 2024 PB 🤭. I'm sure I couldn't crank out ~90 min half's like this back then - but apparently I had an epic race day in 2024 (at Bestseller Aarhus City Half) that is hard for me to beat still 🤔. I hope I get to beat it in 2026 😎👏🏻
+
+# ITRA Stats
+During the year I reached ITRA Advanced 4 level - a milestone that reflects the accumulation of ultra distance races and vertical gain. The journey to this level has taken me through adventures in Austria (Mozart 100), Sweden (Österlen Spring Trail and Kullamannen), and various trail races in Denmark.
+
```

Comment:
Split the images table in 2. First one has the 2 first images. 

Move sentence starting with "The journey..." to after this table. Append to that section a sentence: " Below you see further race stats and a screenshot of the ITRA 'legend' table describing the point ranges that applies to each level". 

Then comes the Second table (has the next 4).

(verify all text for correct spelling and proper use)